### PR TITLE
Jakarta JSON Processing 1.1

### PR DESCRIPTION
**Pull request by m0mus**
Signed-off-by: Dmitry Kornilov <dmitry.kornilov@oracle.com>

- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat_x.y.pdf
- [x] Specification HTML in the form of wombat_x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://oss.sonatype.org/content/groups/staging/jakarta/json/jakarta.json-api/1.1.6/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/staged/eclipse-jsonp-tck-1.1.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/jsonp/issues/204

Include the following in PR#2:
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

**Comment by dblevins** [comment]
Spec PDF looks good.

File: jsonp/1.1/_index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+# Compatible Implementations
+
+* Eclipse JSON Processing 1.1.6
```
Comment (bshannon) [inline jsonp/1.1/_index.md]: Should include a link to the implementation project.

File: jsonp/1.1/tck_results_summary.txt [hunk 2]
```diff
@@ -0,0 +1,1450 @@
+[javatest.batch] Number of Tests Passed      = 169
```
Comment (bshannon) [inline jsonp/1.1/tck_results_summary.txt]: The TCK results should be published on a page related to the implementation project,
not here.

File: jsonp/1.1/_index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Jakarta JSON Processing Specification 1.1"
+date: 2019-07-19T11:00:00+0100
```
Comment (bshannon) [inline jsonp/1.1/_index.md]: The time component is not needed, and probably not useful in this context.

File: jsonp/1.1/_index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+summary: "First release for Jakarta EE 8"
+---
+Jakarta JSON Processing defines a Java(R) based framework for parsing, generating, transforming, and
+querying JSON documents.
+
+* [Jakarta JSON Processing 1.1 Specification Document](./jsonp_1.1.pdf) (PDF)
+* [Jakarta JSON Processing 1.1 Specification Document](./jsonp_1.1.html) (HTML)
+* [Jakarta JSON Processing 1.1 Javadoc](./apidocs)
+* [Jakarta JSON Processing 1.1 TCK](http://download.eclipse.org/ee4j/jsonp/jsonp-tck/jsonp-tck-eftl/release/eclipse-jsonptck-1.0.0.zip)
```
Comment (bshannon) [inline jsonp/1.1/_index.md]: This should be the URL of the location where the signed version will be published:
download.eclipse.org/jakartaee/jsonp/1.1/eclipse-jsonp-tck-1.1.0.zip

File: jsonp/1.1/_index.md (hunk 3 above)
Comment (kwsutter) [reply jsonp/1.1/_index.md]: We should probably update the template so that we can remove this time component comment...

Comment (bshannon) [reply jsonp/1.1/_index.md]: Already updated: https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md

**Comment by dblevins** [comment]
Note I yanked this text as the new requirement is a certification request for the impl:

```
- [x] The URL of the TCK results summary:
```

**Comment by starksm64** [comment]
For wave 1, this spec should be due by 8/7, so please review this ASAP. Note there is a good checklist for https://github.com/jakartaee/specification-committee/blob/master/spec_review_checklist.md

File: jsonp/1.1/_index.md (hunk 1 above)
Comment (m0mus) [reply jsonp/1.1/_index.md]: Done

File: jsonp/1.1/tck_results_summary.txt (hunk 2 above)
Comment (m0mus) [reply jsonp/1.1/tck_results_summary.txt]: Done

File: jsonp/1.1/_index.md (hunk 3 above)
Comment (m0mus) [reply jsonp/1.1/_index.md]: Done

File: jsonp/1.1/_index.md (hunk 4 above)
Comment (m0mus) [reply jsonp/1.1/_index.md]: Fixed

**Comment by starksm64** [comment]
# Spec Review Checklist

1. Spec PR
  - [x] PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
  - [x] Directory of form {spec}/x.y
  - [x] PDF of form {spec}-spec-x.y.pdf ("-spec" preferred but not required)
  - [x] HTML of form {spec}-spec-x.y.html ("-spec" preferred but not required)
  - [x] Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
  - [x] Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
  - [x] No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
  - [x] Staging repository link of the form https://oss.sonatype.org/conte/repositories/staging/jakarta/{spec}/jakarta-{spec}-api/x.y.z/
  - [x] EFTL TCK link of the form http://download.eclipse.org/.../+.zip
  - [x] Compatibility certification link of the form https://github.com/eclipse-ee4j/{project}/#{issue}
  - [x] Second PR for just apidocs

2. _index.md
  - [x] Link to spec pdf
  - [x] Link to spec html
  - [x] Link to apidocs
  - [x] Link to final TCK download zip file of the form https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip
  - [x] Link to API jar file of the form https://search.maven.org/artifact/jakarta.{spec}/jakarta.{spec}-api/x.y.z/jar
  - [x] Name of and link to at least one Compatible Implementation

3. javadocs
  - [x] Footer contains Eclipse copyright and link to license
  - [x] ESFL license is included, usually as doc-files/speclicense.html
  - [x] no META-INF directory in PR

4. Spec PDF
  - [x] Correct spec title
  - [x] Version number of the form x.y, not x.y.z
  - [x] Correct Eclipse copyright line
  - [x] No DRAFT or SNAPSHOT

5. Spec HTML
  - [x] Same as PDF

6. TCK zip file
  - [ ] README file (optional for this release)
  - [x] EFTL license file, preferably named LICENSE.md
  - [x] User's Guide (or equivalent documentation)
  - [x] How to test the Compatible Implementation(s) listed in _index.md above with the TCK (may be in UG)

7. TCK User's Guide (or equivalent documentation)
  - [x] Software requirements listed
  - [x] Installation and configuration described
  - [x] How to run tests
  - [x] Where to file challenges

8. Compatibility certification request
  - [ ] Request follows [template](https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md)
  - [ ] SHA-256 fingerprint matches staged TCK zip file
  - [ ] Request issue has `certification` label.

9. TCK results summary
  - [ ] Page is hosted by Compatible Implementation project
  - [ ] Includes all information from certification request
  - [ ] Summary includes number of tests passed, failed, errors

**Comment by starksm64** [comment]
The link to the proposed TCK is not working, can you verify that @m0mus

**Comment by dblevins** [comment]
@m0mus The staged API binaries will be expired and deleted automatically by Nexus OSSRH on Sunday, Sept 1 st.  Can you publish this api jar to Maven Central at your earliest convenience?

**Comment by dblevins** [comment]
Note there is an spec PDF in this repository.  This should be dropped rather than published as we are not putting the EFSL specs in Maven Central.

 - https://oss.sonatype.org/content/repositories/jakartajson-1045/

**Comment by starksm64** [comment]
- [x] The designated specification committee representative adds this final checklist to the main PR.
 - [ ] The specification committee representative merges the specification PRs.
 - [ ] The designated specification committee representative calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [ ] The specification project member who created the staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript.
 - [ ] The EMO updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [ ] The specification project team should approve the compatibility request and send an email to tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [ ] The specification project team should merge any final release branch as approriate for the branch management for the project.

**Comment by m0mus** [comment]
- [x] The specification project member who created the staging release promotes the specification api jars to maven central.
- [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
- [x] The specification project team should approve the compatibility request and send an email to tck@eclipse.org for approval of the compatible implementation for trademark usage.
- [x] The specification project team should merge any final release branch as approriate for the branch management for the project.

Link to apidocs returns 404. I checked the sources. The link `./apidocs` is fine. Looks like the staged version of the [web site I am looking at](https://deploy-preview-401--jakartaee.netlify.com/specifications/jsonp/1.1/) doesn't have apidocs commit merged.

**Pull request merged**